src/test/java/org/cactoos/map/BehavesAsMap.java

```diff
@@ -69,27 +67,9 @@ public boolean matchesSafely(final Map<K, V> map) {
             map,
             new IsMapContaining<>(
                 new IsEqual<>(this.key),
-                new IsAnything<>()
-            )
-        );
-        MatcherAssert.assertThat(
-            "Doesn't contain the value",
-            map,
-            new IsMapContaining<>(
-                new IsAnything<>(),
                 new IsEqual<>(this.value)
             )
         );
-        MatcherAssert.assertThat(
-            "Doesn't contain the key in #keySet()",
-            map.keySet(),
-            new IsCollectionContaining<>(new IsEqual<>(this.key))
-        );
-        MatcherAssert.assertThat(
-            "Doesn't contain the value in #values()",
-            map.values(),
-            new IsCollectionContaining<>(new IsEqual<>(this.value))
```

Comment:
@Umbrah why remove this check?

Reply:
I wanted this class to check for a key-value pair in a map but not to check for it separately(as it was previously). Do you think we should leave checking for a value in values?